ospfd/ospf_lsa.c

```diff
@@ -2670,9 +2670,8 @@ struct ospf_lsa *ospf_lsa_install(struct ospf *ospf, struct ospf_interface *oi,
 
 			if (IS_DEBUG_OSPF(lsa, LSA_REFRESH)) {
 				zlog_debug(
-					"ospf_lsa_install() Premature Aging lsa 0x%p, seqnum 0x%x",
-					(void *)lsa,
-					ntohl(lsa->data->ls_seqnum));
+					"ospf_lsa_install() Premature Aging lsa %p, seqnum 0x%x",
+					lsa, ntohl(lsa->data->ls_seqnum));
```

Comment:
Please use `__func__`  instead of the name of the function

Reply:
Done. Thanks for your review :)